relay-base-schema/src/data_category.rs

```diff
@@ -166,6 +166,7 @@ impl DataCategory {
             Self::Profile => "profile",
             Self::ProfileIndexed => "profile_indexed",
             Self::Replay => "replay",
+            Self::DoNotUseReplayVideo => "do_not_use_replay_video",
```

Comment:
We need to keep this the same, it's public interface. If someone still uses the category as a string etc. it will break some stuff.

We can only rename the code symbol.